it/uk/gov/hmrc/gatekeeper/support/DeveloperServiceStub.scala

```diff
@@ -82,4 +94,43 @@ trait DeveloperServiceStub {
           .withBody(Json.toJson(users).toString())
       ))
   }
+
+  def primeDeveloperServiceEmailPreferencesBySelectedTopicPaginated(users: Seq[RegisteredUser], topic: TopicOptionChoice, offset: Int = 0, limit: Int = 15): Unit = {
+    val topicParam       = s"topic=${topic.toString}"
+    val paginationParams = s"offset=$offset&limit=$limit"
+
+    val emailPreferencesByTopicUrl = s"$emailPreferencesPaginatedUrl?$topicParam&$paginationParams"
+    stubFor(get(urlEqualTo(emailPreferencesByTopicUrl))
+      .willReturn(
+        aResponse()
+          .withStatus(Status.OK)
+          .withBody(Json.toJson(UserPaginatedResponse(users.size, users.toList)).toString())
+      ))
+  }
+
+  def primeDeveloperServiceEmailPreferencesBySelectedSubscribedApisPaginated(users: Seq[RegisteredUser], service: Seq[String], offset: Int = 0, limit: Int = 15): Unit = {
+    val serviceParam     = s"service=${service.head}"
+    val paginationParams = s"offset=$offset&limit=$limit"
+
+    val emailPreferencesByTopicUrl = s"$emailPreferencesPaginatedUrl?$serviceParam&$paginationParams"
```

Comment:
Did you mean to call this `emailPreferencesBySelectedSubscribedApiUrl`?

Reply:
Addressed